src/Faithlife.Data/DbConnector.cs

```diff
@@ -132,6 +132,29 @@ public abstract class DbConnector : IDisposable, IAsyncDisposable
 		/// <param name="parameters">The command parameters.</param>
 		public DbConnectorCommand Command(string text, params (string Name, object? Value)[] parameters) => new DbConnectorCommand(this, text, DbParameters.Create(parameters));
 
+		/// <summary>
+		/// Creates a new command.
+		/// </summary>
+		/// <param name="text">The text of the command.</param>
+		/// <param name="isStoredProcedure">Whether or not this command is a stored procedure.</param>
+		public DbConnectorCommand Command(string text, bool isStoredProcedure) => new DbConnectorCommand(this, text, default, isStoredProcedure);
```

Comment:
```suggestion
		public DbConnectorCommand StoredProcedure(string text) => new DbConnectorCommand(this, text, default, isStoredProcedure);
```

Per the issue, I'd prefer this syntax.